TFT/src/User/Menu/ScreenSettings.c

```diff
@@ -391,6 +393,12 @@ void menuBrightnessSettings(void)
         listViewRefreshItem(curIndex);
         break;
 
+      case 3:
+        infoSettings.block_touch_on_idle = (infoSettings.block_touch_on_idle + 1) % 2;
```

Comment:
infoSettings.block_touch_on_idle = 1 - infoSettings.block_touch_on_idle

Reply:
I followed the syntax used all around the project (also already present in ScreenSettings). Although that solution is even more compact

Reply:
Which one do you think is faster? I have a hunch that the bitwise operation...

Reply:
In the project, many times a boolean variable is compared with value 1 (for the true value) so it should be better to avoid the use of XOR or !. The more safe expression is the current one or b = (1 - b)

Reply:
Just as a fun fact, an if..then is 10 times faster than %,
`i++;`
`if (i==circular_rollover_limit) i=0;`
 is 10 times faster than
`i = (i + 1) % circular_rollover_limit;`

Division always takes a hit on MCU.